modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -1546,6 +1551,36 @@ private long checkPoolStarvation(
                 EventType.EVT_NODE_JOINED, localNode());
 
         startTimer.finishGlobalStage("Await exchange");
+
+        if (log.isInfoEnabled())
+            ackExpPlcInfoByCaches();
+    }
+
+    /**
+     * Acks expiration policy info by caches.
+     */
+    private void ackExpPlcInfoByCaches() {
```

Comment:
func name confusing, change it for more informative.
Also we need tests checks correct logging all cache expiration variations.